modules/heat_transfer/include/kernels/ADJouleHeatingSource.h

```diff
@@ -12,8 +12,32 @@
 #include "ADKernelValue.h"
 
 /**
- * This kernel calculates the heat source term corresponding to joule heating,
- * Q = J * E = elec_cond * grad_phi * grad_phi, where phi is the electrical potential.
+ *  This kernel supplies the heat source term corresponding to joule heating (Q).
+ *  The residual is provided by the 'ElectromagneticHeatingMaterial' object and
+ *  can be structured in the following formulations:
+ *
+ *  Time domain formulation:
+ *    Q = (conductivity * E) * E,
+ *    - where E is the time domain electric field.
+ *
+ *  Frequency domain formulation:
+ *    Q = 0.5 Re( conductivity * E * E^* ),
+ *    - where E is the real component of the frequency domain electric field,
+ *    - and E^* is the complex conjugate of the electric field.
+ *
+ *  In the case of the time domain formulation, the electric field can be
+ *  defined using an electromagnetic solver or using the electrostatic
+ *  approximation, where:
+ *    E = gradient(V)
+ *    - where V is the electrostatic potential
```

Comment:
These are good equations to have to explain the code. Put the equations also in the documentation

Reply:
I have add these equations to the `ADJouleHeatingSource` documentation.